specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/applications.json

```diff
@@ -0,0 +1,403 @@
+            },
+            "put": {
+                "tags": [
+                    "Applications"
+                ],
+                "description": "Creates applications for the HDInsight cluster.",
+                "operationId": "Applications_Create",
+                "x-ms-examples": {
+                    "Create Application": {
+                        "$ref": "./examples/CreateApplication.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApplicationNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ApplicationGetProperties"
+                        },
+                        "description": "The application create request."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK. The request has succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/Application"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "Applications"
+                ],
+                "description": "Deletes the specified application on the HDInsight cluster.",
+                "operationId": "Applications_Delete",
+                "x-ms-examples": {
+                    "Delete Application from HDInsight cluster": {
+                        "$ref": "./examples/DeleteApplication.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApplicationNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {

```

Comment:
Since this is an LRO - does it also support 202 status code?

Reply:
Correct.